docs/cloud-providers/add-cloud-providers-credentials.md

```diff
@@ -13,6 +13,16 @@ links:
     featured:
 ---
 
+# Generating Cloud Providers Credentials
+## GCP Service Account Credentials
+1. After creating your project, got to GCP IAM manager and select the `Service Accounts` option on the left menu
+2. If you already have a service account, jump to step 5
```

Comment:
Perhaps only deviate from the regular step by step flow when something is missing?
```suggestion
2. If you do not have a service account, follow [instructions on GCP docs to create a service account key](https://cloud.google.com/iam/docs/creating-managing-service-account-keys)
```